src/styles.scss

```diff
@@ -1,176 +1,238 @@
-// Generic styles
-.align {
-  &--center {
-    text-align: center;
+// Add root id to have higher CSS specificity for local overrides
+#root {
+  min-width: 365px;
+  overflow-x: auto !important;
+
+  // Generic styles
+  .align {
+    &--center {
+      text-align: center;
+    }
+
+    &--right {
+      text-align: right;
+    }
   }
 
-  &--right {
-    text-align: right;
+  .Landing-container {
+    padding-top: 4rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
   }
-}
 
-.Landing-container {
-  padding-top: 4rem;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
+  .Landing-buttons-wrapper {
+    padding: 1.5rem 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
 
-.Landing-buttons-wrapper {
-  padding: 1.5rem 0;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+    & > a {
+      margin-bottom: 2rem;
+    }
+  }
 
-  & > a {
-    margin-bottom: 2rem;
+  .WalletButtons-container {
+    width: 100%;
+    max-width: 590px;
```

Comment:
should this be in `rem`?

Reply:
I wouldn't mind keeping these in `px` because it's easier to match them to the window size (for media queries, etc). I haven't noticed any issues with scaling when using `px` like this.